github/ipld.yml

```diff
@@ -2707,10 +2639,6 @@ repositories:
     collaborators:
       admin:
         - Stebalien
-      push:
-        - dvc94ch
```

Comment:
dvc94ch was the original creator, we then moved this repo. In good open source spirit, someone should at least talk with him about it and not just removing push access.